pom.xml

```diff
@@ -67,18 +67,24 @@
     </licenses>
 
     <properties>
-        <maven-compiler-plugin.version>3.2</maven-compiler-plugin.version>
+        <!-- Java version -->
```

Comment:
Seems like we should have a spals parent pom that includes all of our common libraries we like to use. Thoughts?